src/api/types.ts

```diff
@@ -0,0 +1,45 @@
+import { Static, Type } from '@sinclair/typebox';
+
+export const SmartContractRegEx =
+  /[0123456789ABCDEFGHJKMNPQRSTVWXYZ]{28,41}\.[a-zA-Z]([a-zA-Z0-9]|[-_]){0,39}/;
+
+export const TokenQuerystringParams = Type.Object({
+  locale: Type.Optional(
+    Type.String({ description: 'Metadata localization to retrieve', examples: ['es-MX', 'jp'] })
+  ),
+});
+
+export const MetadataAttribute = Type.Object({
+  trait_type: Type.String(),
+  value: Type.Any(),
```

Comment:
Any reason for this `any` type?

Reply:
Good catch. I created this PR to mitigate this problem by specifying the metadata value types https://github.com/hirosystems/token-metadata-service/pull/72

However, I don't think we'll be able to completely get rid of using `any` because SIP-016 specifies a "recursive" type for these values in which the `Array` can contain many values within itself:
```
{"anyOf": [{"type": "object"}, {"type": "string"}, {"type": "number"}, {"type": "integer"}, {"type": "boolean"}, {"type": "array"}]}
```
Typebox supports type recursivity but only for `Object` and we need a `Union` here. I had to leave the `Array` type with an `Any` inside to fix this problem, but let me know if you have a better idea 🙏